### ISSUE TITLE
ecsta seems to require unnecessary permission `ecs:DiscoverPollEndpoint`

### ISSUE THREAD

**Issue by snaka** [fujiwara/ecsta#74]
**Expected behavior:**

`ecsta exec` should execute commands with the same IAM permission as `aws ecs execute-command`.

**Actual behavior:**

When running `ecsta exec` with an IAM user that can successfully execute `aws ecs execute-command` , the following error occurs:

```
User: arn:aws:iam::xxxxxxxx:user/foo is not authorized to perform: ecs:DiscoverPollEndpoint
 on resource: * because no identity-based policy allows the ecs:DiscoverPollEndpoint action
```

This indicates that ecsta requires the `ecs:DiscoverPoolEndpoint` permission, which does not seem necessary based on the AWS CLI's behavior.

**Background:**

I am trying to use ecsta to execute commands in my ECS tasks. I have an IAM user with permissions to execute commands using the AWS CLI, but ecsta requires additional permissions that seems unnecessary. This may be an unnecessary permission requirement.

**Comment by fujiwara** [comment]
@snaka, Thank you for the report!

I made a mistake in resolving an endpoint of ECS. I fixed it by #75.

Could you try v0.6.2? https://github.com/fujiwara/ecsta/releases/tag/v0.6.2

**Comment by snaka** [comment]
@fujiwara 

Thanks!
It works!
